gremlin-go/driver/performance-tests/java-test/src/main/java/PerformanceTest.java

```diff
@@ -0,0 +1,170 @@
+        executeGetToListPerformanceTest();
+        System.out.println("~~~~~~~ PERFORMANCE TESTS COMPLETE ~~~~~~~");
+    }
+
+    public static TimingData executeGetNextPerformanceTest() {
+        final List<Duration> durations = new ArrayList<>();
+        for (int i = 0; i < SAMPLE_SIZE; i++) {
+            try {
+                final String[] args = getArgs(VALUE_MAP_REPEATS);
+                final Instant starts = Instant.now();
+                // execute and retrieve(timed)
+                getProjectTraversal(g, VALUE_MAP_REPEATS, args).next();
+                final Instant ends = Instant.now();
+                durations.add(Duration.between(starts, ends));
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+        TimingData data = getTimingDataFromDurationList(durations);
+        System.out.println(data.toStringMillis("One Item"));
+        return data;
+    }
+
+    public static TimingData executeGetToListPerformanceTest() {
+        final List<Duration> durations = new ArrayList<>();
+        for (int i = 0; i < SAMPLE_SIZE; i++) {
+            try {
+                final String[] args = getArgs(VALUE_MAP_REPEATS);
+                final Instant starts = Instant.now();
+                // execute and retrieve(timed)
+                getProjectTraversal(g, VALUE_MAP_REPEATS, args).toList();
+                final Instant ends = Instant.now();
+                durations.add(Duration.between(starts, ends));
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+        TimingData data = getTimingDataFromDurationList(durations);
+        System.out.println(data.toStringMillis("List"));
+        return data;
+    }
+
+    private static TimingData getTimingDataFromDurationList(final List<Duration> durations) {
+        Collections.sort(durations);
+        durations.remove(durations.size()-1);
+        return new TimingData(
+                durations.stream().reduce(Duration.ZERO, Duration::plus).dividedBy(SAMPLE_SIZE),
+                durations.get(durations.size() / 2),
+                durations.get((int)(durations.size() * 0.90)),
+                durations.get((int)(durations.size() * 0.95)),
+                durations.get(0),
+                durations.get(durations.size() - 1)
+        );
+    }
+}
+
+class TimingData {
+    final Duration AVG;
+    final Duration MEDIAN;
+    final Duration PERCENTILE_90;
+    final Duration PERCENTILE_95;
+    final Duration MIN;
+    final Duration MAX;
+
+    TimingData(Duration avg, Duration median, Duration percentile_90, Duration percentile_95, Duration min, Duration max) {
+        AVG = avg;
+        MEDIAN = median;
+        PERCENTILE_90 = percentile_90;
+        PERCENTILE_95 = percentile_95;
+        MIN = min;
+        MAX = max;
+    }
+
+    @Override
+    public String toString() {
+        return toStringMillis("unknown");
```

Comment:
This seems like an oversight? Was this supposed to take something in to describe the test?

Reply:
its more of a placeholder. usage in the code is toStringMillis with test names passed.

Reply:
I'd say we just remove toString(), don't need it, doesn't add anything either.